### PR TITLE
Update Orature repository urls & Rename env vars

### PR DESCRIPTION
**Pull request by AnonymousWalker**
- Update link to resource container (Orature) repository
- Changes to env var name for better comprehension

This change is [Reviewable](https://reviewable.io/reviews/bible-translation-tools/fetcher/106)

### CODE REVIEW

**Review by danparisd — DISMISSED** [review]

File: fetcher-pipeline/gl_repo_urls.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ https://content.bibletranslationtools.org/WA-Catalog/pt-br_ulb
 https://content.bibletranslationtools.org/WA-Catalog/my_ulb
 https://content.bibletranslationtools.org/WA-Catalog/ceb_ulb
 https://content.bibletranslationtools.org/WA-Catalog/zh_ulb
-https://content.bibletranslationtools.org/WA-Catalog/en_ulb
+https://content.bibletranslationtools.org/WycliffeAssociates/en_ulb
```
Comment (AnonymousWalker) [inline fetcher-pipeline/gl_repo_urls.txt:9]: Note: we can update this once WACS en_ulb has the new content
https://content.bibletranslationtools.org/WA-Catalog/en_ulb

**Review by danparisd — APPROVED** [review]

**Review by mXaln — COMMENTED** [review]
> Reviewed 3 of 3 files at r1.
> *[Reviewable](https://reviewable.io/reviews/bible-translation-tools/fetcher/106)* status: all files reviewed, 1 unresolved discussion (waiting on @jsarabia and @PurpleGuitar)

___
*[fetcher-pipeline/gl_repo_urls.txt, line 9 at r1](https://reviewable.io/reviews/bible-translation-tools/fetcher/106#-MUiVGuN4QXAR6_ArzXJ-r1-9:-MUjC8ki2s2f5nL-Vbho:byetebj) ([raw file](https://github.com/bible-translation-tools/fetcher/blob/8419b59a193d531a57e35f31a770f123e23d4c10/fetcher-pipeline/gl_repo_urls.txt#L9)):*
<details><summary><i>Previously, AnonymousWalker (Tony) wrote…</i></summary><blockquote>

Note: we can update this once WACS en_ulb has the new content
https://content.bibletranslationtools.org/WA-Catalog/en_ulb
</blockquote></details>

I'm hesitant about this. I think WA-Catalog has approved and published resources, when WycliffeAssociates is in active editing. I might be wrong. Maybe @purpleguitar knows better about this. And I think we should use resources from one organization, not from different.

**Comment by AnonymousWalker** [comment]
___
*[fetcher-pipeline/gl_repo_urls.txt, line 9 at r1](https://reviewable.io/reviews/bible-translation-tools/fetcher/106#-MUiVGuN4QXAR6_ArzXJ-r1-9:-MUjDz_93oYHUXKkq4c1:b-qyvpw9) ([raw file](https://github.com/bible-translation-tools/fetcher/blob/8419b59a193d531a57e35f31a770f123e23d4c10/fetcher-pipeline/gl_repo_urls.txt#L9)):*
<details><summary><i>Previously, mXaln (Maxim) wrote…</i></summary><blockquote>

I'm hesitant about this. I think WA-Catalog has approved and published resources, when WycliffeAssociates is in active editing. I might be wrong. Maybe @purpleguitar knows better about this. And I think we should use resources from one organization, not from different.
</blockquote></details>

I would say we may have it now for the purpose of testing, which we can change later on when WACS has such content.

**Review by mXaln — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/bible-translation-tools/fetcher/106#-:-MUjFu4Z8Mi3CMyuYNWO:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @jsarabia and @PurpleGuitar)

**Pull request merged**